spek-samples/src/test/kotlin/org/jetbrains/spek/samples/subject/CalculatorTableSpec.kt

```diff
@@ -0,0 +1,38 @@
+package org.jetbrains.spek.samples.subject
+
+import org.jetbrains.samples.Calculator
+import org.jetbrains.spek.api.Spek
+import org.jetbrains.spek.api.dsl.describe
+import org.jetbrains.spek.api.dsl.it
+import org.jetbrains.spek.api.dsl.on
+import org.jetbrains.spek.table.example
+import org.jetbrains.spek.table.with
+import kotlin.test.assertEquals
+
+object CalculatorTableSpec : Spek({
+    describe("subtract") {
+        with(
+            example(4, 2),
```

Comment:
You can set exception as expected *value*, catch actual and compare with expected.